rep-0003.rst

```diff
@@ -205,7 +250,7 @@ While we mainly develop with gcc, no use of compiler-specific features is allowe
 without proper use of macros to allow use on other platforms.
 
 Use of C++11/C++14 features and filesystem/networking/etc... TS's (Technical Specifications) is allowed if they are checked for at configure time and equivalent functionality can be provided with the extra compiler features.
-We will continue using only C++03 features to preserve backwards compatibility, but code must also compile with C++11 compilers, i.e. when ``-std=c++11`` is used.
+Support for C++11 is now a compiler requirement, but the API will not use any C++11-specific feature.
```

Comment:
Yeah, these are the rules we should follow for the core packages (historically `desktop-full`). For everyone else they are just guidelines which they may follow if they agree with the rational.